test/test_ripper_tags.rb

```diff
@@ -815,4 +823,27 @@ class Foo
     assert_equal 'Foo', tags[0][:name]
     assert_equal 'calculate', tags[1][:name]
   end
+
+  def test_method_defined_in_block
+    tags = extract(<<-EOC)
+      RSpec.describe do
+        def test_method; end
+      end
+
+      module M
+        included do
+          def included_method; end
+          attr_accessor :cache
+        end
+
+        %w[sub gsub].each do |m|
+          define_method("\#{m}!") {}
```

Comment:
We shouldn't generate a tag for such a method, right? In other words, the [failing test](https://github.com/tmm1/ripper-tags/actions/runs/5012861065/jobs/8985301886?pr=105#step:4:23) correctly informs us of the error.

Reply:
Yep, this is something that needs to be fixes. I believe this test was added by @mislav [here](https://github.com/tmm1/ripper-tags/pull/105/commits/8e034e8a6082a5121cec079de9857bca2938d631).